### PR TITLE
improve validation for aliases

### PR DESCRIPTION
**Pull request by generall**
- Check if alias exists on collection creation
- Force delete alias on collection deletion
- Tests

### CODE REVIEW

**Review by IvanPleshkov — APPROVED** [review]

File: openapi/tests/openapi_integration/test_alias.py [hunk 1]
```diff
@@ -13,6 +13,78 @@ def setup():
     drop_collection(collection_name=collection_name)
 
 
+def test_aliases_corner_cases():
```
Comment (ffuugoo) [inline openapi/tests/openapi_integration/test_alias.py:16]:
```suggestion
def test_cant_create_alias_if_collection_exists():
```

File: openapi/tests/openapi_integration/test_alias.py [hunk 2]
```diff
@@ -13,6 +13,78 @@ def setup():
...
+                    }
+                }
+            ]
+        }
+    )
+    assert not response.ok
+    assert response.status_code == 400
+
+
+def test_create_collection_after_alias():
```
Comment (ffuugoo) [inline openapi/tests/openapi_integration/test_alias.py:39]:
```suggestion
def test_cant_create_collection_if_alias_exists():
```

**Review by ffuugoo — COMMENTED** [review]
- Should we also add `test_cant_create_alias_if_alias_exists`? (Or does such test already exist?)
- Should we also test, that if we remove an aliased collection, the alias is also removed, and we can successfully create another collection with the same name then?

**Comment by generall** [comment]
> Should we also add test_cant_create_alias_if_alias_exists? (Or does such test already exist?)

This should result in success, idempotency

**Pull request merged**